ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -149,6 +149,22 @@ tests:
       EC2_AMI: ami-0addfb94c944af1cc
       EC2_INSTANCE_TYPE: c6g.metal
     workflow: openshift-microshift-e2e-metal-tests
+- as: metal-bootc-periodic-test
```

Comment:
These jobs look the same as already existing ones (`ocp-metal-nightly` for example), what is the difference with them?

Reply:
Just saw the difference, ignore my comment.